ruby/lib/minitest/queue.rb

```diff
@@ -102,6 +102,21 @@ def mark_as_flaked!
     end
   end
 
+  module TimeDetails
```

Comment:
[nitpick] Maybe something `WithTimestamps` makes it more clear than `TimeDetails`?